custom/icds_reports/views.py

```diff
@@ -1624,13 +1624,13 @@ def page_context(self):
     def section_url(self):
         return
 
-    @staticmethod
-    @memoized
-    def transifex(domain, form_data):
-        transifex_project_slug = form_data.get('transifex_project_slug')
-        source_language_code = form_data.get('target_lang') or form_data.get('source_lang')
-        return Transifex(domain, form_data['app_id'], source_language_code, transifex_project_slug,
-                         form_data['version'])
+    def transifex(self, domain, form_data):
+        if not hasattr(self, '_transifex'):
+            transifex_project_slug = form_data.get('transifex_project_slug')
+            source_language_code = form_data.get('target_lang') or form_data.get('source_lang')
+            self._transifex = Transifex(domain, form_data['app_id'], source_language_code, transifex_project_slug,
+                                        form_data['version'])
+        return self._transifex
```

Comment:
Looks like this is only accessed directly once, so you could just rename it `get_transifex` and not bother with memoizing the result.